src/commands/sensor/commands/create.handler.ts

```diff
@@ -1,23 +1,33 @@
 import { CreateSensorCommand } from './create.command';
 import { SensorAggregate } from '../aggregates/sensor.aggregate';
 import { SensorRepository } from '../repositories/sensor.repository';
+import { OwnerRepository } from '../../owner/repositories/owner.repository';
 import { ICommandHandler, EventPublisher, CommandHandler } from '@nestjs/cqrs';
+import { NonExistingOwnerException } from '../errors/non-existing-owner-exception';
 import { SensorAlreadyExistsException } from '../errors/sensor-already-exists-exception';
 
 @CommandHandler(CreateSensorCommand)
 export class CreateSensorCommandHandler implements ICommandHandler<CreateSensorCommand> {
   constructor(
     private readonly publisher: EventPublisher,
-    private readonly repository: SensorRepository,
+    private readonly sensorRepository: SensorRepository,
+    private readonly ownerRepository: OwnerRepository,
   ) {}
 
   async execute(command: CreateSensorCommand): Promise<void> {
     let aggregate: SensorAggregate;
-    aggregate = await this.repository.get(command.sensorId);
+    aggregate = await this.sensorRepository.get(command.sensorId);
 
     if (!!aggregate) {
       throw new SensorAlreadyExistsException(command.sensorId);
     } else {
+      if (command.ownerIds) {
```

Comment:
Dit past prima in een aparte util, denk ik? Nu is het copy/paste van dezelfde code, niet?

Reply:
Lijkt mij een goed idee!